### PR TITLE
Add assert statements

### PR DESCRIPTION
**Pull request by Shantaa**
Addition of assert statements to previous changes to HelpWindow

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -50,6 +50,12 @@ public HelpWindow(Stage root) {
         this.contentManager = new HelpContentManager();
         this.logicManager = new HelpLogicManager(contentManager);
         initializeUiComponents();
+
+        assert contentManager != null : "HelpContentManager should be initialized";
+        assert logicManager != null : "HelpLogicManager should be initialized";
+        assert tocListView != null : "Table of Contents ListView should be initialized";
+        assert commandSummaryTable != null : "Command Summary Table should be initialized";
+        assert helpContentFlow != null : "HelpContentFlow should be initialized";
```
Comment (AlphaPyke) [inline src/main/java/seedu/address/ui/HelpWindow.java:58]: lgtm, asserts are important here to ensure instances are initialised

File: src/main/java/seedu/address/ui/HelpContentManager.java [hunk 2]
```diff
@@ -212,21 +222,14 @@ public String getContent(String key) {
...
-                "Marking a Person as Paid/Unpaid",
-                "Saving and Editing Data",
-                "Exiting the Program",
-                "Command Summary"
-        );
+        ObservableList<String> tocList = FXCollections.observableArrayList(
+                "Introduction", "Quick Start", "Command Format Guidelines", "Adding a Contact",
+                "Listing All Contacts", "Editing a Contact", "Finding Contacts", "Deleting a Contact",
+                "Clearing All Entries", "Marking a Person as Paid/Unpaid", "Saving and Editing Data",
+                "Exiting the Program", "Command Summary");
```
Comment (AlphaPyke) [inline src/main/java/seedu/address/ui/HelpContentManager.java:229]: why the change in code design? thought the earlier version might make it clearer to see

File: src/main/java/seedu/address/ui/HelpContentManager.java [hunk 3]
```diff
@@ -16,8 +16,12 @@ public class HelpContentManager {
     private Map<String, String> contentMap = new HashMap<>();
     private final ObservableList<String[]> commandSummaryData = FXCollections.observableArrayList();
 
+    /**
+     * Constructs a HelpContentManager and initializes the content map with predefined help content.
+     */
     public HelpContentManager() {
         initializeContents();
+        assert !contentMap.isEmpty() : "Content map should be initialized with values";
```
Comment (fymbc) [inline src/main/java/seedu/address/ui/HelpContentManager.java:24]: LGTM, good assertions.
